index.js

```diff
@@ -3,9 +3,9 @@ import SVGO from 'svgo';
 import compiler from 'vue-template-compiler';
 import transpile from 'vue-template-es2015-compiler';
 
-const optimizeSvg = (content, config) => new Promise((resolve, reject) => {
+const optimizeSvg = (content, config, path) => new Promise((resolve, reject) => {
   const svgo = new SVGO(config);
-  svgo.optimize(content).then((result) => {
+  svgo.optimize(content, { path }).then((result) => {
```

Comment:
See [SVGO line #24](https://github.com/svg/svgo/blob/72db8ebbbf5aa3ac82a32c5b744cb09deeba5e35/lib/svgo.js#L24) for optimize method.

And [SVGO line #70](https://github.com/svg/svgo/blob/72db8ebbbf5aa3ac82a32c5b744cb09deeba5e35/lib/svgo.js#L70 ) for where the info object (`{ path }`) is passed to plugins.